Pod/Classes/SEGFlurryIntegration.m

```diff
@@ -80,4 +85,18 @@ - (BOOL)screenTracksEvents
     return [(NSNumber *)[self.settings objectForKey:@"screenTracksEvents"] boolValue];
 }
 
+// Returns NSDictionary truncated to 10 entries
+
+-(NSMutableDictionary *)formatProperties:(NSDictionary *) properties
```

Comment:
Instead of formatProperties, should it be `trimProperties` or `truncatProperties`?